site/template.html

```diff
@@ -12,5 +12,7 @@
 <body>
 {content}
 </body>
+<script src='/assembly/clipboard.min.js'></script>
```

Comment:
@davidtheclark can you post a bit more info? copy works for me and it seems to be added to the `dist` directory.